pkg/solver/controller.go

```diff
@@ -520,7 +535,7 @@ func (controller *SolverController) addDeal(ctx context.Context, deal data.Deal)
 	span.SetAttributes(attribute.String("deal.id", deal.ID))
 	span.AddEvent("data.get_deal_id.done")
 
-	controller.log.Info("add deal", deal)
+	controller.log.Info().Any("deal", deal).Msg("add deal")
```

Comment:
Let's also add the dealId and the resourceProviderId of the RP whose going to do the job as key-value pairs here

Reply:
Also done in 157cc4c5175bc5b951e312772925194e59acf480. Did the job creator ID as well.